### DISCUSSION TITLE
Fishpig Magento 1 modules open source / free

### DISCUSSION THREAD

**Discussion by DegrizNet** [OpenMage/magento-lts#3748] [Extensions & Modules]
Hello,

Ben from Fishpig just told me that he decided to put all Magento 1 modules open source / free. 
All modules are listed on page:
https://fishpig.com/magento-1-extensions

I have been using his module on a lot of project over the years and they are great.. so I highly reccomend looking at it.
With lastest Openmage most of them (the one I tested / used) work (the one I tested / used) with few little php 8 fixes.

**Comment by fballiano** [comment]
wow! that's great news!

**Reply by fballiano** [reply]
I've added FishPig and Manadev to https://github.com/fballiano/awesome-openmage/blob/main/README.md#vendors-that-opensourced-all-their-modules

**Reply by ma4nn** [reply]
Meanbee has also Open-sourced some useful Magento 1 Extensions: http://www.meanbee.com/magento-extensions

**Reply by fballiano** [reply]
@ma4nn thanks, I've added Meanbee to the list!

**Comment by addison74** [comment]
All my respect to such developers.

It is a great pity when the work and especially the efforts are thrown in the trash because a good product can no longer be used. Fortunately with some modifications most Magento 1 extensions can be used in OpenMage.

**Reply by sreichel** [reply]
Also wrote to Amasty times ago ... to work at their repos with a NDA to keep modules alive, but got no reply.

Maybe someone wants to offer same?

**Reply by addison74** [reply]
Many top developers removed in the last months their Magento 1 extensions. BSS Commerce is one example.

**Reply by addison74** [reply]
There is no need to contact Amasty. They still offer the old extensions here https://amasty.com/magento-extensions.html.

**Reply by fballiano** [reply]
i've also contacted BSS, let's see

**Reply by sreichel** [reply]
@ADDISON74 the extension are not updated anymore - as far as i know. (php8 support, features ...)

**Reply by addison74** [reply]
@sreichel - There are many providers in this situation, they no longer offer support, but those extensions from Amasty are still for sale.

**Reply by sreichel** [reply]
They might work with vanilla M1, but not with OM ... thats a problem (for this projects LTS).

Same for fishpig, and all others. Nice to have them opensource, but these modules have to be maintained.

**Reply by fballiano** [reply]
with all my customers I'm on OM20.3 with many of those modules and I haven't had many problems (with warnings off), some quick patches here and there and that's it. it will be a different game with php9 but at the moment...
